storage/innobase/handler/ha_innodb.cc

```diff
@@ -4955,6 +4959,9 @@ static bool dd_open_hardcoded(space_id_t space_id, const char *filename) {
   }
 
   if (fail) {
+    ib::error(ER_XB_MSG_2)
+        << "If mysql.ibd opening failed with flags mismatch, try startup with"
+        << " innodb_encrypt_tables = ON/OFF";
```

Comment:
Can you rephrase, when to try ON and when to try OFF?

Reply:
Done